src/editor.html

```diff
@@ -889,6 +889,7 @@
 					zss_editor.calculateEditorHeightWithCaretPosition();
 					zss_editor.setScrollPosition();
 					zss_editor.enabledEditingItems(e);
+					notifyIfSelectedTextHasChanged();
```

Comment:
Could it be that we should do this before updating the (selected) items via `zss_editor.enabledEditingItems`? Otherwise the current toolbar's content might flicker 🤔